### ISSUE TITLE
Add kustomize files to all operators

### ISSUE THREAD

**Issue by lfrancke** [stackabletech/issues#46]

**Comment by teozkr** [comment]
Do we really want to support both this and #45?

**Comment by lfrancke** [comment]
Yeah, I believe this will be a one-time effort and will offer more possibilities for deployment.
I see no real downsides, the overhead should be minimal.

You think different?
